Statistics/Histograms/Histogram.cs

```diff
@@ -115,10 +115,7 @@ public Histogram(double[] data, double binWidth)
             _BinWidth = binWidth;
             if (data == null)
             {
-                Min = 0;
-                Max = Min + _BinWidth;
-                Int64 numberOfBins = 1;
-                _BinCounts = new Int32[numberOfBins];
+                throw new ArgumentNullException("The histogram cannot be constructed without data. Use a different constructor.");
```

Comment:
i dont think we need to throw the argumentnullexception. i would rather not create exceptions in construtors. id change the if to if data!=null, do what is in the else case right now, else, dont do anything.

Reply:
we might set _bincounts = new int32[]

Reply:
done